ambry-store/src/main/java/com/github/ambry/store/CompactionManager.java

```diff
@@ -288,6 +288,9 @@ public void run() {
                     metrics.markCompactionStart(true);
                     compactionStarted = true;
                     store.compact(details, bundleReadBuffer);
+                    if (storeConfig.storeAutoCloseLastLogSegmentEnabled && store.getReplicaId().isSealed()) {
+                      store.closeLastLogSegmentIfQualified();
+                    }
```

Comment:
This comes after `store.compact()` because it's intended for next round of compaction? 
I am trying to understand if the ordering matters.